python/src/opendp/v1/_lib.py

```diff
@@ -10,31 +10,26 @@
     'bool': ['bool']
 }
 
-
-def _get_lib_dir() -> str:
+lib = None
+if os.environ.get('OPENDP_HEADLESS', "false") == "false":
     lib_dir = os.environ.get("OPENDP_LIB_DIR", os.path.join(os.path.dirname(os.path.abspath(__file__)), "lib"))
     if not os.path.exists(lib_dir):
         # fall back to default location of binaries in a developer install
         build_dir = 'debug' if os.environ.get('OPENDP_TEST_RELEASE', "false") == "false" else 'release'
         lib_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), *['..'] * 4, 'rust', 'target', build_dir)
     if not os.path.exists(lib_dir):
         raise ValueError("Unable to find lib directory. Consider setting OPENDP_LIB_DIR to a valid directory.")
-    return lib_dir
-
 
-def _get_lib_name() -> str:
     platform_to_name = {
         "darwin": "libopendp_ffi.dylib",
         "linux": "libopendp_ffi.so",
         "win32": "opendp_ffi.dll",
     }
     if sys.platform not in platform_to_name:
         raise Exception("Platform not supported", sys.platform)
-    return platform_to_name[sys.platform]
-
+    lib_name = platform_to_name[sys.platform]
 
-lib_path = os.path.join(_get_lib_dir(), _get_lib_name())
-lib = ctypes.cdll.LoadLibrary(lib_path)
+    lib = ctypes.cdll.LoadLibrary(os.path.join(lib_dir, lib_name))
```

Comment:
Something seems funky here. Are these changes intended, or is this a merge issue?

Reply:
This was something I added. When the docs build, it sources the library. If the binaries don't exist, the sourcing fails, so the docs fail to build. So he had to start building the rust binaries from the docs workflow to get the docs to build. This changes it so that if the env var `OPENDP_HEADLESS` is set, then the binaries are not loaded.